### ISSUE TITLE
Playback error in Safari

### ISSUE THREAD

**Issue by Tuphal** [toniebox-reverse-engineering/teddycloud#119]
When accessing the webinterface and wanting to play a Tonie, it is not working.

An `Unhandled Promise Rejection: NotSupportedError: The operation is not supported.` error is triggered in the brower console as soon as I click the play button.
I uploaded an mp3 to test a different source and to exclude ogg extension as issue. When inspecting and injecting following source in the fileviewer playback element, it is playing without issue
`<audio controls="" style="width: 0px; visibility: hidden; transition: 1s;" src="/content/13B9E00F/**test/03 - Rommel-Bommel.mp3**?ogg=true&amp;special=content"></audio>`

When using Firefox to view the files, its content plays without issues.

Seems as Safari does not ogg extensions?
Ans possibility for an workaround?

**Comment by SciLor** [comment]
It looks like Safari does not fully support OPUS at the moment.

https://caniuse.com/opus

**Comment by b2un0** [comment]
had the same issue.

is is possible to remux the content on the fly into `.webm` with ffmpeg?

**Comment by SciLor** [comment]
Yes, this would be a possible approach. Either repackaging the OGG OPUS into WEBM OPUS by hand or by using ffmpeg.js
I think this would be something for the frontend to do, as this is a specific Safari problem.
